include/freerdp/window.h

```diff
@@ -153,14 +153,14 @@ typedef struct _WINDOW_ORDER_INFO WINDOW_ORDER_INFO;
 
 struct _ICON_INFO
 {
-	UINT32 cacheEntry;
-	UINT32 cacheId;
-	UINT32 bpp;
-	UINT32 width;
-	UINT32 height;
-	UINT32 cbColorTable;
-	UINT32 cbBitsMask;
-	UINT32 cbBitsColor;
+	UINT16 cacheEntry;
```

Comment:
While this is the correct value range size type it will break public API, so keep the old one.